### DISCUSSION TITLE
繁中服涤火杰西卡宿舍识别问题

### DISCUSSION THREAD

**Discussion by windhug** [MaaAssistantArknights/MaaAssistantArknights#10670] [General]
想知道有没有其他繁中服的朋友也和我一样
涤火杰西卡在中枢可以被识别但是在宿舍却不行
![屏幕截图 2024-09-23 131052](https://github.com/user-attachments/assets/23ffcbe4-882e-413e-a1d4-7f61fb50f9a9)
![屏幕截图 2024-09-23 131132](https://github.com/user-attachments/assets/6ad96883-9ce5-42a9-914a-d2ac02181729)

**Comment by ABA2396** [comment]
应该是识别错了，跑一遍任务然后发个日志看看

**Comment by Daydreamer114** [comment]
请前往主仓库提交 issue
